easybuild/easyconfigs/h/HAPGEN2/HAPGEN2-2.2.0.eb

```diff
@@ -0,0 +1,25 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = 'Tarball'
+
+name = 'HAPGEN2'
+version = '2.2.0'
+
+homepage = 'https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/hapgen2.html'
+description = """'HAPGEN2' simulates case control datasets at SNP markers."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://mathgen.stats.ox.ac.uk/genetics_software/hapgen/download/builds/x86_64/v%(version)s/']
+sources = ['hapgen2_x86_64.tar.gz']
```

Comment:
@rtripath89 same here, if the version is not included in the name of the source tarball, please include a checksum too